File: 4/solution.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number}
 */
var findMedianSortedArrays = function(nums1, nums2) {
    // step 1 and step 2
    const combined = [... nums1, ... nums2].sort()
    // step 3
    if (combined.length % 2 === 1)
        return combined[Math.floor(combined.length/2)] 
    else
        return (
            combined[Math.floor(combined.length/2)] + (combined[Math.floor(combined.length/2)-1]))/2
    }

File: 4/solution.py
class Solution(object):
    def findMedianSortedArrays(self, nums1, nums2):
        """
        :type nums1: List[int]
        :type nums2: List[int]
        :rtype: float
        """
        # step 1 and step 2: combine and sort arrays
        combined = sorted(nums1 + nums2)
        # step 3: check if even or odd # of values
        if len(combined) % 2 == 1: 
            print(len(combined) // 2)
            return combined[len(combined) // 2]
        else:
            ans = float((combined[len(combined) // 2] + combined[len(combined) // 2 - 1]))
            return ans / 2